optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/ranking/ResilientScoreComparator.java

```diff
@@ -20,11 +20,18 @@
 import java.util.Comparator;
 
 import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.score.definition.ScoreDefinition;
 
 /**
  * Able to compare {@link Score}s of different types or nulls.
  */
-public class ResilientScoreComparator implements Comparator<Score>, Serializable {
+final class ResilientScoreComparator implements Comparator<Score>, Serializable {
+
+    private final ScoreDefinition aScoreDefinition;
+
+    public ResilientScoreComparator(ScoreDefinition aScoreDefinition) {
```

Comment:
Are we sure about this?
Most users don't know about ScoreDefinition.
This means most users can never use a ResilientScoreComparator.
But on the other hand, I presume they will never need it, so it's fine?

Reply:
This is no longer a public class. And even before now, it was not in public API.
We are fine.